### DISCUSSION TITLE
proper setup

### DISCUSSION THREAD

**Discussion by xbtgit** [stephenasuncionDEV/nfthost#51] [Q&A] [answered]
Hi @stephenasuncionDEV I wanted to get your help if possible.  I'm really dumb and I can't figure out how to setup the following for 

NFThost:

1. How do I setup the ENCRYPT_KEY & CREATE_WEBSITE_TOKEN variables in the .env file for the frontend build ? 

2. How do I setup the ACCESS_TOKEN_SECRET
REFRESH_TOKEN_SECRET
THIRDPARTY_TOKEN_SECRET in the backend server .env file?

Bro please help me out I'm trying to understand how I can deploy the frontend and backend on localhost with proper variables and configuration

**Comment by stephenasuncionDEV** [comment, marked as answer]
@wordpress-is-not-a-backend-my-dude 

You can set it up like the following:

ENCRYPT_KEY= could be any hash string
CREATE_WEBSITE_TOKEN= to generate, check the commented out code here: https://github.com/stephenasuncionDEV/nfthost-backend/blob/main/index.js

ACCESS_TOKEN_SECRET= could be any hash string
REFRESH_TOKEN_SECRET= could be any hash string
THIRDPARTY_TOKEN_SECRET= could be any hash string

**Comment by khanfar** [comment]
hi , i try website in my phone , but looks like not connect to my metamask in phone , is there something im missing ??? i purchase the premium package . but the same issue . anyone help please ???

**Comment by AnthonyFon** [comment]
> Hi @stephenasuncionDEV I wanted to get your help if possible. I'm really dumb and I can't figure out how to setup the following for
> 
> NFThost:
> 
> 1. How do I setup the ENCRYPT_KEY & CREATE_WEBSITE_TOKEN variables in the .env file for the frontend build ?
> 2. How do I setup the ACCESS_TOKEN_SECRET
>    REFRESH_TOKEN_SECRET
>    THIRDPARTY_TOKEN_SECRET in the backend server .env file?
> 
> Bro please help me out I'm trying to understand how I can deploy the frontend and backend on localhost with proper variables and configuration

Hi @wordpress-is-not-a-backend-my-dude ,

I'm trying to deploy this project, but always fail. Have you success ?
Can you help me ?
@stephenasuncionDEV can you help me too ?

Thx a lot.
Best regards
